charts/cinder-csi-plugin/values.yaml

```diff
@@ -5,32 +5,32 @@ csi:
   attacher:
     image:
       repository: quay.io/k8scsi/csi-attacher
-      tag: v2.0.0
+      tag: v2.2.0
       pullPolicy: IfNotPresent
   provisioner:
     image:
       repository: quay.io/k8scsi/csi-provisioner
-      tag: v1.4.0
+      tag: v1.6.0
       pullPolicy: IfNotPresent
   snapshotter:
     image:
       repository: quay.io/k8scsi/csi-snapshotter
-      tag: v1.2.2
+      tag: v2.1.1
```

Comment:
https://github.com/kubernetes/cloud-provider-openstack/pull/974
please check this , it's not reasonable to do directly update

Reply:
I'll keep this simple then and bump to what exists in https://github.com/kubernetes/cloud-provider-openstack/tree/release-1.18/manifests/cinder-csi-plugin

Please update the helm chart in #974 so that we have both example manifests and the helm chart aligned.

Reply:
ok, will take further look at this, thanks